wordpress-mobile/WordPress-iOS
# Enable UI tests on Gutenberg releases

This PR enables automatic execution of our UI test suite on Gutenberg release branches. This means that Gutenberg release PRs will now benefit from `EditorGutenbergTests`–as well as the other tests in our suite–without developers needing to remember to run these tests.

If this change is approved we'll create a matching PR for WordPress-Android.

h/t to @ceyhun for first suggesting automating this, after [we added to the Gutenberg mobile release checklist](https://github.com/wordpress-mobile/release-toolkit-gutenberg-mobile/pull/31) a step to trigger the execution of these tests.

To test:
1. Notice that the branch of this PR follows the Gutenberg release branch naming convention i.e. `gutenberg/integrate_release_X.XX.X`, so we can expect GitHub to now automatically run UI tests on this branch
2. Verify that UI tests are automatically run

## Regression Notes
Not applicable since this isn't a user-facing change.

PR submission checklist:
